gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -67,113 +57,63 @@
  *   {@link org.apache.gobblin.runtime.Task}(s), it persists the {@link TaskState} of each {@link org.apache.gobblin.runtime.Task} to
  *   a file that will be collected by the {@link GobblinHelixJobLauncher} later upon completion of the job.
  * </p>
- *
- * @author Yinan Li
  */
 @Alpha
 public class GobblinHelixTask implements Task {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(GobblinHelixTask.class);
+  private static final Logger _logger = LoggerFactory.getLogger(GobblinHelixTask.class);
```

Comment:
Why the change here? We usually avoid mixing '_' and this.

Reply:
The existing code doesn't use this consistently anyway. 
This change is made to make logger usage more consistent and fit the google coding style and Linkedin coding style.

Reply:
Replaced all the fields with _ prefix and removed all usages of this.

Reply:
I don't see a consistent usage of this.logger elsewhere if the convention is not to mix _ with this. Did I miss something?